Functional.py

```diff
@@ -0,0 +1,89 @@
+
+# # 1
+
+def sequential_map(*args):
+    l = len(args) - 1
+    c = args[-1]
+    for i in args[0:l]:
+        c = i(c)
+    return list(c)
+
+
+# # 2
+
+def consensus_filter(*args):
+    l = len(args) - 1
+    c = args[-1]
+    
+    for i in args[0:l]:
+        cnt = -1
+        res = [i(x) for x in c]
+        for j in res:
+            cnt += 1
+            if not j:
+                c.remove(c[cnt])
+                cnt -= 1
```

Comment:
Очень костыльно, функциональное программирование не используется
```suggestion
  *functions, container = args
  for func in functions:
      container = filter(func, container)
  return list(container) 
```